samcli/lib/build/app_builder.py

```diff
@@ -634,6 +645,8 @@ def _build_function_in_process(
         if lambda_builders_version == "1.8.0":
```

Comment:
Can we also change this to `1.9.0` for now?

Reply:
What's purpose for this if statement?

Reply:
We need this until we release new version of lambda builders. Otherwise pylint fail because these new parameters are not available there.

Reply:
Updated.